jupyterhub/files/hub/jupyterhub_config.py

```diff
@@ -293,6 +293,16 @@ def camelCaseify(s):
         if cfg_key is None:
             cfg_key = camelCaseify(trait)
         set_config_if_not_none(c.GitLabOAuthenticator, trait, 'auth.gitlab.' + cfg_key)
+elif auth_type == 'azuread':
+    c.JupyterHub.authenticator_class = 'azuread'
```

Comment:
Looking at other options, I think this will be wrong, and that it should be for example:

```suggestion
    c.JupyterHub.authenticator_class = 'oauthenticator.azuread.AzureAdOAuthenticator'
```

Reply:
Actually, no, this is correct. After JupyterHub 1.0, authenticator classes can be specified via entrypoint alias, 'azuread' or 'github' or 'gitlab'. I was adopting the new style for the new option, but maybe it's better to adopt the new style all at once, so I'll make this one match here and do that update separately.